bitnami/grafana/values.yaml

```diff
@@ -186,6 +186,13 @@ dashboardsConfigMaps: []
 datasources:
   secretName: ""
 
+## Create notifiers from a configMap
+## The notifiersName must contain the files
+## @param notifiers.notifiersName Notifiers name containing custom configMap files
+##
+notifiers:
+  notifiersName: ""
```

Comment:
This parameter should be named `notifiers.configMapName` to follow the same format we used for Grafana dashboard provider or datasources:
```suggestion
## @param notifiers.configMapName Name of a ConfigMap containing Grafana notifiers configuration
##
notifiers:
  configMapName: ""
```

Reply:
Thank you for your suggestion. I've changed the name of the parameter and bumped the minor version.